src/xdocs/config_whitespace.xml

```diff
@@ -2018,15 +2018,27 @@ public @interface Beta {} // empty annotation type
         <p>
         may optionally be exempted from the policy using the <code>
         allowEmptyMethods</code>, <code>allowEmptyConstructors
-        </code>, <code>allowEmptyTypes</code>, <code>allowEmptyLoops</code> and
-        <code>allowEmptyLambdas</code> properties.
+        </code>, <code>allowEmptyTypes</code>, <code>allowEmptyLoops</code>
+        <code>allowEmptyLambdas</code> and <code>allowEmptyCatches</code>
+        properties.
         </p>
         <p>This check does not flag as violation double brace initialization like:</p>
         <pre><code>
 new Properties() {{
     setProperty("key", "value");
 }};
         </code></pre>
+        <p>Parameter allowEmptyCatches allows to suppress violations when token
+        list contains SLIST to check if beginning of block is surrounded by
```

Comment:
I don't think we should mention `SLIST` or token types in description documentation. User may not be familiar with our tokens and tree structure. It should just explain java code.
We should stick to `catch` blocks and it's braces.